### PR TITLE
allow check-supervisor to connect with login and password

### PR DESCRIPTION
**Pull request by luxifer**

### CODE REVIEW

File: bin/check-supervisor.rb [hunk 1]
```diff
@@ -40,6 +40,16 @@ class CheckSupervisor < Sensu::Plugin::Check::CLI
          long: '--port PORT',
          default: 9001
 
+  option :username,
+         description:   'Supervisor HTTP username',
```
Comment (analytically) [inline bin/check-supervisor.rb]: Please follow the indentation style of the file.

Comment (luxifer) [inline bin/check-supervisor.rb]: OK. I just copied these options from the `check-supervisor-socket.rb`

**Comment by luxifer** [comment]
Is it OK?

**Comment by eheydrick** [comment]
@luxifer Looks good. Can you add an entry to the CHANGELOG under unreleased.

**Comment by luxifer** [comment]
I'll do that
Le 22 févr. 2016 6:35 PM, "Eric Heydrick" notifications@github.com a
écrit :

**Comment by luxifer** [comment]
Done

**Comment by luxifer** [comment]
Sorry to up but did you have time to check this ?

**Comment by luxifer** [comment]
Hi, is ot possible to tag a new release with this PR? Thanks in advance

**Pull request merged**